src/QA/QA.Web/Controllers/DashboardController.cs

```diff
@@ -17,9 +20,26 @@ public DashboardController(DashboardOrchestrator orchestrator)
         [HttpGet("/", Name = RouteNames.Dashboard_Index_Get)]
         public async Task<IActionResult> Index()
         {
-            var vm = await _orchestrator.GetDashboardViewModelAsync();
+            var vm = await _orchestrator.GetDashboardViewModelAsync().ConfigureAwait(false);
```

Comment:
I don't think we need to use ConfigureAwait in Core. https://stackoverflow.com/questions/13489065/best-practice-to-call-configureawait-for-all-server-side-code